src/commands/deploy.ts

```diff
@@ -539,6 +533,83 @@ Additionally, you can also retry the build with the debug flag:
     }
   }
 
+  async __detectPlatformVersion(config: any, body: any) {
```

Comment:
Let's use `if...return` structure instead of `if...else`. It makes the code more readable. For example:

```js
if(x) {
  // ...
  return body;
}

if(y) {
  // ...
  return body;
}

// ...
return body;
```

Reply:
I don't understand. It's switch case. So people will get it that only one statement is going to be executed.

Reply:
When there's a `return` statement, you don't need to read the following lines. You're sure that this is the end of this block.
But with else and switch statements, you have to always read the whole thing because the execution will continue to the end of the block.

Reply:
https://stackoverflow.com/a/9267684

Reply:
https://medium.com/swlh/return-early-pattern-3d18a41bba8

Reply:
It's called: Early return pattern.

Reply:
So, instead of using "return," we just go with "break," and when you see that, it's like a sign telling you to stop looking. Any particular lines bugging you?

Reply:
OK. You didn't mean the switch case I think.